processor/spanmetricsprocessor/processor.go

```diff
@@ -32,17 +33,19 @@ import (
 	"go.uber.org/zap"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanmetricsprocessor/internal/cache"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanmetricsprocessor/keybuilder"
 )
 
 const (
-	serviceNameKey     = conventions.AttributeServiceName
-	operationKey       = "operation"   // OpenTelemetry non-standard constant.
-	spanKindKey        = "span.kind"   // OpenTelemetry non-standard constant.
-	statusCodeKey      = "status.code" // OpenTelemetry non-standard constant.
-	metricKeySeparator = string(byte(0))
-	traceIDKey         = "trace_id"
-
-	defaultDimensionsCacheSize = 1000
+	serviceNameKey             = conventions.AttributeServiceName
+	instrumentationLibraryName = "spanmetricsprocessor"
+	operationKey               = "operation"   // OpenTelemetry non-standard constant.
```

Comment:
As @Tenaria mentioned it's a non-standard constant in OTel, I pressume it's not published by the collector.

Reply:
They used to be defined as constants (if you look through the history) but OTel removed them somewhere along the way. It might be worth looking into why it was removed in a future PR. 

p.s the status code is the status code on the span (OK, ERROR, UNSET) not stuff specific to http